notebooks/test_first.ipynb

```diff
@@ -138,9 +746,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Did you mean to include cell outputs in this commit? It will be better clear all cell output to avoid bloating the repo.

(This will stop being an issue once we do #15 :smile: -- I can prioritize that next if interested)

Reply:
I did an interactive rebase and chose to reword the commits, but now I'm seeing a second set of them tracked in this PR... guessing I didn't fix that correctly

Reply:
Interactive rebase will change the commit hashes, so that's correct actually!
You would have to force-push the new commits to replace the previous commit history:
```
git push --force-with-lease
```

Regarding removing cell outputs: adding an additional commit that removes the cell output still leaves the output in the commit history, which continues to bloat the repo. What's needed ultimately is to change that commit that introduces the output and remove the output from it. Alternatively, the two commits (the one that introduces cell output and the one that removes it) can be squashed to achieve that same effect. Happy to hop onto a call if I can help in any way!

Reply:
(It looks like you did force-push so that's good.)